language/move-prover/bytecode/src/stackless_structured_control_flow.rs

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) The Diem Core Contributors
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::stackifier_topological_sort::StackifierTopologicalSort;
+use crate::stackless_control_flow_graph::StacklessControlFlowGraph;
+
+pub struct StacklessStructuredControlFlow {
+    pub top_sort: StackifierTopologicalSort,
```

Comment:
I think this representation of the CFG is not sufficient. You somehow need to encode conditionals. Because you aren't doing this, also the generated output doesn't deal correctly with if-then-else.

Reply:
@wrwg Thanks for the review, after the last commit, as shown in "ControlStructures.exp.apply-cfg-to-scf", the YUL codes are still using the `switch`, and the YUL errors are resolved now.
```
!! Succeeded compiling Yul
```

I might need some help to emit `do-while` and `if-else` instead, can we have a quick chat when you are free, maybe tomorrow?